SpanishFilter/sections/specific.txt

```diff
@@ -31,6 +31,7 @@ todopolicia.com##.samBannerUnit
 /tghr.js$script,redirect=noopjs,domain=serieslatinoamerica.tv
 ||hentaistube.com/hentaistubepop.js
 primedeportes.es###yuio
+lanacion.com.ar##main > .lay-sidebar > aside.sidebar__aside
```

Comment:
I'm not sure we should remove it, 
`lanacion.com.ar#$#.sidebar__aside { background: none!important; }` to annoyances maybe